app/assets/stylesheets/_commons.css.scss

```diff
@@ -222,3 +222,25 @@ body {
   .tab-bar .menu-icon { text-indent: 0.6rem; }
   #squiggle { fill: $bright-green; }
 }
+
+.social-icons {
+  a + a { margin-left: 1rem; }
+
+  i {
+    background: $bright-green;
+    border-radius: 35px;
+    color: white;
+    font-size: 3rem;
+    height: 56px;
+    line-height: 1.25;
+    text-align: center;
+    width: 56px;
```

Comment:
Properties should be ordered background, border-radius, color, font-size, height, line-height, text-align, transition, width